ci/pipelines/pull-request/jinja.template.yml

```diff
@@ -66,6 +67,140 @@ resource_types:
     tag: latest
     username: ((!docker-username))
 jobs:
+- name: {{build_test.name}}
```

Comment:
Is there a plan to DRY this up soon?  Wholesale duplication of hundreds of line of code where the only actual difference is the gradle args leaves a bad taste.